test-runner/src/main/scala/com/typesafe/spark/test/mesos/framework/runners/MesosIntegrationTestRunner.scala

```diff
@@ -4,14 +4,20 @@ import com.typesafe.config.ConfigFactory
 import com.typesafe.spark.test.mesos.framework.runners.Utils._
 
 object MesosIntegrationTestRunner {
-
-
   def main(args: Array[String]): Unit = {
-
     implicit val config = ConfigFactory.load()
     printMsg(config.toString)
 
-    val failures: Int = ClientModeRunner.run(args) + ClusterModeRunner.run(args) + MultiClusterModeRunner.run(args)
+    // TODO(tnachen): Add support with a args parsing library to avoid
+    // parsing args based on positions and support flags.
+    val withZk = args.exists(a => a.equals("--with-zk"))
```

Comment:
I prefer one type of check whether we are using zk. The one based on the zk uri check here:
https://github.com/typesafehub/mesos-spark-integration-tests/blob/master/test-runner/src/main/scala/com/typesafe/spark/test/mesos/framework/runners/ClientModeRunner.scala#L28
This could be put in a helper method to re-use it in other places too. 
How do you run this suite on DCOS , do you use a zk uri?